dsl/src/main/groovy/org/cristalise/dsl/persistency/outcome/SchemaDelegate.groovy

```diff
@@ -178,8 +178,11 @@ class SchemaDelegate {
                     if (f.dynamicForms.updateQuerytRef != null) updateQuerytRef(f.dynamicForms.updateQuerytRef)
                     if (f.dynamicForms.outOfSpecs != null) {
                         outOfSpecs {
-                            if (f.dynamicForms.outOfSpecs.pattern != null) pattern(f.dynamicForms.outOfSpecs.pattern)
-                            if (f.dynamicForms.outOfSpecs.message != null) message(f.dynamicForms.outOfSpecs.message)
+                            if (f.dynamicForms.outOfSpecs.pattern != null)    pattern(f.dynamicForms.outOfSpecs.pattern)
+                            if (f.dynamicForms.outOfSpecs.message != null)    message(f.dynamicForms.outOfSpecs.message)
+                            if (f.dynamicForms.outOfSpecs.expression != null) expression {
+                                mkp.yieldUnescaped("<![CDATA[ ${f.dynamicForms.outOfSpecs.expression}]]>")
```

Comment:
When is the warning shown? If OutOfSpec.expression evaluates to true, that would tell me that the vaue entered is OutOfSpec, so the warning should be show.

Reply:
I do not undrestand what you are saying....

Reply:
The warning is shown when the expression evaluates to true or when it evaluates to false?

Reply:
I think this requirements is for the webui as it does not effect the functionality of neither  dsl nor outcomebuilder.

Reply:
OK, we can call it Warning.